Engines/Wine/Verbs/mfWorkaround/script.js

```diff
@@ -0,0 +1,113 @@
+const Wine = include("engines.wine.engine.object");
+const Resource = include("utils.functions.net.resource");
+const { Extractor } = include("utils.functions.filesystem.extract");
+const { ls, cp } = include("utils.functions.filesystem.files");
+//const { getGithubReleases } = include("utils.functions.net.githubreleases"); commented out because it's useless until z0z0z will use releases
+
+const Optional = Java.type("java.util.Optional");
+const Regedit = include("engines.wine.plugins.regedit");
+const Regsvr32 = include("engines.wine.plugins.regsvr32");
+const OverrideDLL = include("engines.wine.plugins.override_dll");
+
+/**
+ * Verb to install Media Foundation workaround
+ * see: https://github.com/z0z0z/mf-install
+ */
+class mfWorkaround {
+    constructor(wine) {
+        this.wine = wine;
+    }
+
+    /**
+     * Sets the used mfWorkaround version
+     *
+     * @param {string} mfWorkaroundVersion The version of mfWorkaround to downlaod
+     * @returns {mfWorkaround} The mfWorkaround object
+     */
+    withVersion(mfWorkaroundVersion) {
+        this.mfWorkaroundVersion = mfWorkaroundVersion;
+
+        return this;
+    }
+
+    go() {
+        const wizard = this.wine.wizard();
+        const prefixDirectory = this.wine.prefixDirectory();
+        const system32directory = this.wine.system32directory();
+        const system64directory = this.wine.system64directory();
+
+        //commented out because it's useless until z0z0z will use releases
+        /*if (typeof this.mfWorkaroundVersion !== "string") {
+            const versions = getGithubReleases("Kron4ek", "FAudio-Builds", wizard);
+            this.mfWorkaroundVersion = versions[0];
+        }*/
+
+        const setupFile = new Resource()
+            .wizard(wizard)
+            .url(
+                `https://codeload.github.com/z0z0z/mf-install/zip/master`
+            )
+            .name(`mf-install-master.zip`)
+            .get();
+
+        new Extractor()
+            .wizard(wizard)
+            .archive(setupFile)
+            .to(`${prefixDirectory}/mfWorkaround/`)
+            .extract();
+
+        const mfWorkaroundDir = `${prefixDirectory}/mfWorkaround/mf-install-master`;
+
+        ls(`${mfWorkaroundDir}/syswow64`).forEach(file => {
+            if (file.endsWith(".dll")) {
+                cp(`${mfWorkaroundDir}/syswow64/${file}`, system32directory);
+
+                new OverrideDLL(this.wine).withMode("native", [file]).go();
+            }
+        });
```

Comment:
A valid alternative could be:

```javascript
ls(`${mfWorkaroundDir}/syswow64`)
   .filter(file => file.endsWith(".dll"))
   .forEach(file => {
      cp(`${mfWorkaroundDir}/syswow64/${file}`, system32directory);

      new OverrideDLL(this.wine).withMode("native", [file]).go();
   });
```

Reply:
Why is the current option bad?

Reply:
It is not bad per se. I just wanted to show you an alternative style that you can you but don't need to. This comment is purely optional

Reply:
Understood